Chapter02/Activity2.01/app/src/main/java/com/example/loginactivity/MainActivity.kt

```diff
@@ -25,13 +27,29 @@ class MainActivity : AppCompatActivity() {
     private var isLoggedIn = false
     private var loggedInUser = ""
 
+    private val submitButton by lazy {
```

Comment:
By this point 'by lazy' hasn't been explain, but then I guess the Kotlin code has not been explained in depth so this is fine.

Reply:
Actually 'by lazy' hasn't been explained in the course so please update 'code' in exercise 2.0.3 fo for first - thank you.